game-core/src/main/java/games/strategy/ui/SwingComponents.java

```diff
@@ -147,12 +147,11 @@ public static JTabbedPane newJTabbedPane(final int width, final int height) {
     return tabbedPane;
   }
 
-  public static ButtonGroup newButtonGroup(final JRadioButton... radioButtons) {
+  public static void newButtonGroup(final JRadioButton... radioButtons) {
```

Comment:
Also possibly outside the scope of this PR, but, again, this reads like a factory method but doesn't return anything.  Some ideas: `addToNewButtonGroup()`, `groupButtons()`.

Reply:
`assignToButtonGroup` sound good?   (528f4f3b5)

Reply:
This one is in-scope, method and return value should match, good consideration @ssoloff , thank you.